optimade/grammar/v1.2.0.develop.lark

```diff
@@ -26,15 +30,16 @@ property_first_comparison: property ( value_op_rhs
                                     | known_op_rhs
                                     | fuzzy_string_op_rhs
                                     | set_op_rhs
-                                    | set_zip_op_rhs )
+                                    | set_zip_op_rhs
+                                    | length_op_rhs )
 
-constant_first_comparison: constant value_op_rhs
+constant_first_comparison: constant OPERATOR ( non_string_value | not_implemented_string )
 
 value_op_rhs: OPERATOR value
 known_op_rhs: IS ( KNOWN | UNKNOWN )
-fuzzy_string_op_rhs: CONTAINS string
-                   | STARTS [ WITH ] string
-                   | ENDS [ WITH ] string
+fuzzy_string_op_rhs: CONTAINS value
+                   | STARTS [ WITH ] value
+                   | ENDS [ WITH ] value
```

Comment:
Oh. So it's no longer a fuzzy *string* operator?

Reply:
This is just an annoying git/GitHub issue with the diff. After deleting the old grammars and renaming them, it decided to map v0.10.0 to v1.2 as v1.0.0 was unchanged. The diff between v1.2.0 and v1.1 is just the boolean stuff. I figure this will all get squashed out anyway...

Reply:
Alternatively I could try to rebase this into multiple commits that delete/rename separately...

Reply:
> Alternatively I could try to rebase this into multiple commits that delete/rename separately...

Just done this, not sure what happens if we squash merge (the diff will probably still try to pick up on this weirdness)

Reply:
Of course, we can also split it into two PRs...

Reply:
Ah, then never mind. It's a bit weird still that the explicit "fuzzy string operation" takes in numbers etc. But all good.